lib/openhab/core/types/date_time_type.rb

```diff
@@ -193,9 +209,13 @@ def wday
         #
         # @return [String] The timezone in `[+-]hh:mm(:ss)` format (`Z` for UTC)
         #
-        # @deprecated This method has been deprecated in openHAB 4.3.
```

Comment:
Note: the #zone method is a part of our time conversion / comparison, so I believe it should not be deprecated.